plugins/services/src/js/service-configuration/PodContainerArtifactsConfigSection.js

```diff
@@ -20,21 +20,33 @@ class PodContainerArtifactsConfigSection extends React.Component {
         heading: 'Executable',
         prop: 'executable',
         render(prop, row) {
-          return <BooleanValue options={BOOLEAN_OPTIONS} value={row[prop]} />;
+          return (
+            <ConfigurationMapBooleanValue
```

Comment:
In general we stick to being explicit in our naming.
This is inconsistent with what we do elsewhere, if we are going to do this we should have a general way to do this, so we avoid confusion. I was very confused, when I saw this.
Let's add it to our frontend sync as a topic, e.g. 'How do we get short names, whilst staying explicit?'